src/engine/Engine.sol

```diff
@@ -173,21 +174,16 @@ contract Engine is IEngine, DSMath {
         return IPriceSource(registry.priceSource());
     }
 
-    /// @notice Helper to check whether an address is either a Hub or a Spoke
-    function __isHubOrSpoke(address _who) private view returns (bool) {
-        // Check if hub
-        if (registry.fundIsRegistered(_who)) return true;
-        // Check if spoke
-        else {
-            // 1. Spoke points to hub
-            // 2. Hub confirms it is a spoke
-            // 3. Fund exists for hub
-            try ISpoke(_who).getHub() returns (IHub hub) {
-                return hub.isSpoke(_who) && registry.fundIsRegistered(address(hub));
-            }
-            catch {
-                return false;
-            }
+    /// @notice Helper to check whether an address is a valid fund Vault
```

Comment:
@fubhy Natspec isn't parsed for internal/private functions, so we had decided just to use `@notice` tags for general descriptions of these functions. You're right that it might be more correct to use `@dev`, I'd just want to confirm that newlines without a decorator are parsed as the previous decorator (rather than as `@notice` by default, which is what the docs say happens to undecorated lines).

I'm thinking we can leave these as `@notice` for consistency with our codebase for now, but we can easily change all these in a future PR.